docs/guides/LICENSE_GUIDE.md

```diff
@@ -3,17 +3,21 @@
 Make applicable license conditions transparent.
 
 > [!TIP]
-> More information is available from [GitHub documentation](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository).
+> More information is available from [GitHub documentation][github-documentation].
 
 ## Choosing a license
 
 The following [online guide](https://choosealicense.com/) is an excellent way to pick a license.
```

Comment:
Why is this link not extracted (there are several others, won't mention more of them 😉 )

Reply:
Same reason as https://github.com/SRGSSR/guilde-plateformes-propres/pull/7#discussion_r1488232391